proto/gz/msgs/dynamic_detachable_joint.proto

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Contributor: Adarsh Karan K P
+// Neobotix GmbH
+
+syntax = "proto3";
+
+package gz.msgs;
+
+option java_package = "com.gz.msgs";
+option java_outer_classname = "DynamicDetachableJointProtos";
+
+/// \ingroup gz.msgs
+/// \interface DynamicDetachableJoint
+/// \brief Message for dynamic detachable joint
+
+import "gz/msgs/header.proto";
+
+message AttachDetachRequest
+{
+  /// \brief Name of the child model
+  string child_model_name = 1;
+
+  /// \brief Name of the child link
+  string child_link_name = 2;
+
+  /// \brief Command to attach or detach
+  string command = 3;
+}
+
+message AttachDetachResponse
+{
+  /// \brief True if the operation was successful
+  bool success = 1;
+
+  /// \brief Additional message or error description
+  string message = 2;
+}
```

Comment:
This looks like it could be a generic `Result` type. How about:

in `proto/gz/msgs/result.proto`

```proto

message Result
{
  /// \brief 0 if the operation was successful
  int error_code = 1;
  /// \brief Additional message or error description
  string message = 2;
}
```